### PR TITLE
Validation: Removed Disallowed HTML

### PR DESCRIPTION
**Pull request by dereklh7**
## Category

- [x] Content fix
- [ ] New article

## Related issues

This PR is part of the Validation cleanup efforts.

## What's in this Pull Request?

Removed disallowed HTML tags and attributes.

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[624ba1c](https://github.com/dereklh7/sp-dev-docs/commits/624ba1c82615b8da8d0f664568e1b16351179e35)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/sp-add-ins/add-a-web-part-to-a-page-in-a-sharepoint-hosted-sharepoint-add-in.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/add-a-web-part-to-a-page-in-a-sharepoint-hosted-sharepoint-add-in.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/add-a-web-part-to-a-page-in-a-sharepoint-hosted-sharepoint-add-in?branch=pr-en-us-8164) | [Details](#user-content-ad8bd035b93c436381d1cb6dee993642-7dacdf62)
[docs/sp-add-ins/add-in-permissions-in-sharepoint.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/add-in-permissions-in-sharepoint.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/add-in-permissions-in-sharepoint?branch=pr-en-us-8164) | [Details](#user-content-c939658d258a531894d005872a3eac5c-7dacdf62)
[docs/sp-add-ins/authorization-code-oauth-flow-for-sharepoint-add-ins.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/authorization-code-oauth-flow-for-sharepoint-add-ins.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/authorization-code-oauth-flow-for-sharepoint-add-ins?branch=pr-en-us-8164) | [Details](#user-content-d175fa2380b73c773d066bac05367bd1-7dacdf62)
[docs/sp-add-ins/convert-an-autohosted-sharepoint-add-in-to-a-provider-hosted-add-in.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/convert-an-autohosted-sharepoint-add-in-to-a-provider-hosted-add-in.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/convert-an-autohosted-sharepoint-add-in-to-a-provider-hosted-add-in?branch=pr-en-us-8164) | [Details](#user-content-26312a9bb4f27f19c1c6813f2187b6df-7dacdf62)
[docs/sp-add-ins/create-a-provider-hosted-add-in-that-includes-a-custom-sharepoint-list-and-conte.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/create-a-provider-hosted-add-in-that-includes-a-custom-sharepoint-list-and-conte.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/create-a-provider-hosted-add-in-that-includes-a-custom-sharepoint-list-and-conte?branch=pr-en-us-8164) | [Details](#user-content-abb4dadafb6be77640a5bfa55f0f0eae-7dacdf62)
[docs/sp-add-ins/include-a-web-part-in-a-webpage-on-the-add-in-web.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/include-a-web-part-in-a-webpage-on-the-add-in-web.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/include-a-web-part-in-a-webpage-on-the-add-in-web?branch=pr-en-us-8164) | [Details](#user-content-65635286dfd80f695fdb360e1bd4761c-7dacdf62)
[docs/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index?branch=pr-en-us-8164) | [Details](#user-content-ba230203100e92f385ac75b20af22db5-7dacdf62)
[docs/sp-add-ins/three-ways-to-think-about-design-options-for-sharepoint-add-ins.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/three-ways-to-think-about-design-options-for-sharepoint-add-ins.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/three-ways-to-think-about-design-options-for-sharepoint-add-ins?branch=pr-en-us-8164) | [Details](#user-content-52e0b6fa6b4d7245645119c57be117fe-7dacdf62)
[docs/sp-add-ins/working-with-folders-and-files-with-rest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/working-with-folders-and-files-with-rest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/working-with-folders-and-files-with-rest?branch=pr-en-us-8164) | [Details](#user-content-dceedff3e83e1aa0cecb3ecb2f4e0444-7dacdf62)
[docs/sp-add-ins/working-with-lists-and-list-items-with-rest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/working-with-lists-and-list-items-with-rest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/working-with-lists-and-list-items-with-rest?branch=pr-en-us-8164) | [Details](#user-content-657276220738f24bb18332225061ff2f-7dacdf62)

<a id="ad8bd035b93c436381d1cb6dee993642-7dacdf62"></a>
### [docs/sp-add-ins/add-a-web-part-to-a-page-in-a-sharepoint-hosted-sharepoint-add-in.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/add-a-web-part-to-a-page-in-a-sharepoint-hosted-sharepoint-add-in.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="c939658d258a531894d005872a3eac5c-7dacdf62"></a>
### [docs/sp-add-ins/add-in-permissions-in-sharepoint.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/add-in-permissions-in-sharepoint.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="d175fa2380b73c773d066bac05367bd1-7dacdf62"></a>
### [docs/sp-add-ins/authorization-code-oauth-flow-for-sharepoint-add-ins.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/authorization-code-oauth-flow-for-sharepoint-add-ins.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="26312a9bb4f27f19c1c6813f2187b6df-7dacdf62"></a>
### [docs/sp-add-ins/convert-an-autohosted-sharepoint-add-in-to-a-provider-hosted-add-in.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/convert-an-autohosted-sharepoint-add-in-to-a-provider-hosted-add-in.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="abb4dadafb6be77640a5bfa55f0f0eae-7dacdf62"></a>
### [docs/sp-add-ins/create-a-provider-hosted-add-in-that-includes-a-custom-sharepoint-list-and-conte.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/create-a-provider-hosted-add-in-that-includes-a-custom-sharepoint-list-and-conte.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="65635286dfd80f695fdb360e1bd4761c-7dacdf62"></a>
### [docs/sp-add-ins/include-a-web-part-in-a-webpage-on-the-add-in-web.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/include-a-web-part-in-a-webpage-on-the-add-in-web.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="ba230203100e92f385ac75b20af22db5-7dacdf62"></a>
### [docs/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="52e0b6fa6b4d7245645119c57be117fe-7dacdf62"></a>
### [docs/sp-add-ins/three-ways-to-think-about-design-options-for-sharepoint-add-ins.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/three-ways-to-think-about-design-options-for-sharepoint-add-ins.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="dceedff3e83e1aa0cecb3ecb2f4e0444-7dacdf62"></a>
### [docs/sp-add-ins/working-with-folders-and-files-with-rest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/working-with-folders-and-files-with-rest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="657276220738f24bb18332225061ff2f-7dacdf62"></a>
### [docs/sp-add-ins/working-with-lists-and-list-items-with-rest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/working-with-lists-and-list-items-with-rest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206132004008114-8164/BuildReport?accessString=69aa8a57cfea4707e5d5189750c01c14d08e3c06dfc90fc9c546946e173cda29).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by dereklh7** [comment]
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: docs/sp-add-ins/add-in-permissions-in-sharepoint.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ SharePoint Add-ins that are installed to SharePoint are granted permissions when
  
 When you are developing an add-in or troubleshooting an add-in, there may be occasions when you want to change, or regrant, the permissions of an add-in that has already been installed. You can do so with these steps:
 
-1. Go to `http://<SharePointWebSite>/_layouts/15/AppInv.aspx`, where  _<SharePointWebSite>_ is the URL of the website where the add-in is installed. Be careful not to add any query parameters on the URL. The form you need only appears on this page if the URL is exactly as shown.
+1. Go to `http://<SharePointWebSite>_layouts/15/AppInv.aspx`, where  _\<SharePointWebSite\>_ is the URL of the website where the add-in is installed. Be careful not to add any query parameters on the URL. The form you need only appears on this page if the URL is exactly as shown.
```
Comment (andrewconnell) [inline docs/sp-add-ins/add-in-permissions-in-sharepoint.md:298]: Incorrect URL, the forward slack is correct.

You can escape the `<SharePointWebSite>` with backslashes, or replace the <> wth {}

**Review by andrewconnell — CHANGES_REQUESTED** [review]

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[82bbbf4](https://github.com/dereklh7/sp-dev-docs/commits/82bbbf48f5f3403db4ce72a38f9e769899baa21c)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/sp-add-ins/add-a-web-part-to-a-page-in-a-sharepoint-hosted-sharepoint-add-in.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/add-a-web-part-to-a-page-in-a-sharepoint-hosted-sharepoint-add-in.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/add-a-web-part-to-a-page-in-a-sharepoint-hosted-sharepoint-add-in?branch=pr-en-us-8164) | [Details](#user-content-ad8bd035b93c436381d1cb6dee993642-7d3ca952)
[docs/sp-add-ins/add-in-permissions-in-sharepoint.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/add-in-permissions-in-sharepoint.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/add-in-permissions-in-sharepoint?branch=pr-en-us-8164) | [Details](#user-content-c939658d258a531894d005872a3eac5c-7d3ca952)
[docs/sp-add-ins/convert-an-autohosted-sharepoint-add-in-to-a-provider-hosted-add-in.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/convert-an-autohosted-sharepoint-add-in-to-a-provider-hosted-add-in.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/convert-an-autohosted-sharepoint-add-in-to-a-provider-hosted-add-in?branch=pr-en-us-8164) | [Details](#user-content-26312a9bb4f27f19c1c6813f2187b6df-7d3ca952)
[docs/sp-add-ins/create-a-provider-hosted-add-in-that-includes-a-custom-sharepoint-list-and-conte.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/create-a-provider-hosted-add-in-that-includes-a-custom-sharepoint-list-and-conte.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/create-a-provider-hosted-add-in-that-includes-a-custom-sharepoint-list-and-conte?branch=pr-en-us-8164) | [Details](#user-content-abb4dadafb6be77640a5bfa55f0f0eae-7d3ca952)
[docs/sp-add-ins/include-a-web-part-in-a-webpage-on-the-add-in-web.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/include-a-web-part-in-a-webpage-on-the-add-in-web.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/include-a-web-part-in-a-webpage-on-the-add-in-web?branch=pr-en-us-8164) | [Details](#user-content-65635286dfd80f695fdb360e1bd4761c-7d3ca952)
[docs/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index?branch=pr-en-us-8164) | [Details](#user-content-ba230203100e92f385ac75b20af22db5-7d3ca952)
[docs/sp-add-ins/three-ways-to-think-about-design-options-for-sharepoint-add-ins.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/three-ways-to-think-about-design-options-for-sharepoint-add-ins.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/three-ways-to-think-about-design-options-for-sharepoint-add-ins?branch=pr-en-us-8164) | [Details](#user-content-52e0b6fa6b4d7245645119c57be117fe-7d3ca952)
[docs/sp-add-ins/working-with-folders-and-files-with-rest.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/working-with-folders-and-files-with-rest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/working-with-folders-and-files-with-rest?branch=pr-en-us-8164) | [Details](#user-content-dceedff3e83e1aa0cecb3ecb2f4e0444-7d3ca952)
[docs/sp-add-ins/working-with-lists-and-list-items-with-rest.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/working-with-lists-and-list-items-with-rest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/working-with-lists-and-list-items-with-rest?branch=pr-en-us-8164) | [Details](#user-content-657276220738f24bb18332225061ff2f-7d3ca952)

<a id="ad8bd035b93c436381d1cb6dee993642-7d3ca952"></a>
### [docs/sp-add-ins/add-a-web-part-to-a-page-in-a-sharepoint-hosted-sharepoint-add-in.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/add-a-web-part-to-a-page-in-a-sharepoint-hosted-sharepoint-add-in.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="c939658d258a531894d005872a3eac5c-7d3ca952"></a>
### [docs/sp-add-ins/add-in-permissions-in-sharepoint.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/add-in-permissions-in-sharepoint.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="26312a9bb4f27f19c1c6813f2187b6df-7d3ca952"></a>
### [docs/sp-add-ins/convert-an-autohosted-sharepoint-add-in-to-a-provider-hosted-add-in.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/convert-an-autohosted-sharepoint-add-in-to-a-provider-hosted-add-in.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="abb4dadafb6be77640a5bfa55f0f0eae-7d3ca952"></a>
### [docs/sp-add-ins/create-a-provider-hosted-add-in-that-includes-a-custom-sharepoint-list-and-conte.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/create-a-provider-hosted-add-in-that-includes-a-custom-sharepoint-list-and-conte.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="65635286dfd80f695fdb360e1bd4761c-7d3ca952"></a>
### [docs/sp-add-ins/include-a-web-part-in-a-webpage-on-the-add-in-web.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/include-a-web-part-in-a-webpage-on-the-add-in-web.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="ba230203100e92f385ac75b20af22db5-7d3ca952"></a>
### [docs/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="52e0b6fa6b4d7245645119c57be117fe-7d3ca952"></a>
### [docs/sp-add-ins/three-ways-to-think-about-design-options-for-sharepoint-add-ins.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/three-ways-to-think-about-design-options-for-sharepoint-add-ins.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="dceedff3e83e1aa0cecb3ecb2f4e0444-7d3ca952"></a>
### [docs/sp-add-ins/working-with-folders-and-files-with-rest.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/working-with-folders-and-files-with-rest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="657276220738f24bb18332225061ff2f-7d3ca952"></a>
### [docs/sp-add-ins/working-with-lists-and-list-items-with-rest.md](https://github.com/dereklh7/sp-dev-docs/blob/Revert-6-13-2/docs/sp-add-ins/working-with-lists-and-list-items-with-rest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206271457149591-8164/BuildReport?accessString=c56e0506a59f403b9b8a2b8a067a21d9fc6addc361ca89a0712c25f239e6c428).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by andrewconnell — APPROVED** [review]

**Pull request merged**